src/libYARP_dev/src/PolyDriver.cpp

```diff
@@ -214,13 +214,17 @@ bool PolyDriver::coreOpen(yarp::os::Searchable& prop) {
     if (prop.check("device",part)) {
         str = part->toString().c_str();
     }
+
+#ifndef YARP_NO_DEPRECATE // since yarp 2.3.70
```

Comment:
Should be `YARP_NO_DEPRECATED` (a `D` is missing)
Since you should already change this line, please make also "YARP" uppercase in the comment...